### PR TITLE
Added more packages.

### PR DESCRIPTION
**Pull request by adikou**
Imperative that Oh-My-ZSH be the last install since it changes shell and all installations beyond that are moot. Tested on a VM that ran Ubuntu 14.04.

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -67,14 +67,13 @@ function setupSwap() {
 
 function installSSH() {
     echo "--------------------------------------------"
-    echo "Installing SSH..."
-    sudo apt-get install openssh-client
+    # echo "Installing SSH..."
+    # sudo apt-get install openssh-client
     read -p "Generating SSH keys. Enter email address to be associated: " email
```
Comment (onstash) [inline setup.sh:72]: It would make sense to try using git's global user email. If it is not present, then ask for the email.

File: setup.sh [hunk 2]
```diff
@@ -127,10 +126,10 @@ function runSetup() {
         setupPython
         setupOpenInTerminal
         setupSublime
-        setupZsh
         installJava
         installSSH
         initGit
```
Comment (onstash) [inline setup.sh:131]: It would make sense to call `initGit` just after `setupGit`.

File: setup.sh [hunk 3]
```diff
@@ -147,6 +146,16 @@ function runSetup() {
         setupSystemPackages
     elif [ $1 == "python" ]; then
         setupPython
+    elif [ $1 == "java" ]; then
+        installJava
+    elif [ $1 == "git-init" ]; then
+        initGit
+    elif [ $1 == "flash" ]; then
+        installFlash
+    elif [ $1 == "ssh" ]; then
```
Comment (onstash) [inline setup.sh:155]: What package is this - `flash`?

File: setup.sh (hunk 1 above)
Comment (adikou) [inline setup.sh:72]: True. I set this up with the user setting up SSH keys to add to GitHub in mind. In fact, just ssh-keygen would do the trick.

**Pull request merged**